docs/source/install.rst

```diff
@@ -89,6 +89,14 @@ source.
      - s390x
      - :ref:`tier-4`
      - Distributions compatible with the `manylinux 2014`_ packaging specification
+   * - Linux (musl)
+     - x86_64
+     - :ref:`tier-3`
+     -
+   * - Linux (musl)
+     - aarch64
+     - :ref:`tier-3`
```

Comment:
I'll say I don't think supporting this architecture now is a priority. NumPy themselves (https://pypi.org/project/numpy/1.26.0/#files) don't support musl with aarch64 as of now, so the user already needs to have a C compiler anyway, I think it's fair to assume users are "advanced" and have to install a Rust compiler

Reply:
Ah ok, yeah I'll add the exclude for it then. I just assumed it would be in the same class as x86_64 but didn't look.